Goobi/src/de/sub/goobi/helper/GoobiScript.java

```diff
@@ -265,6 +267,30 @@ private void deleteMetadataDirectory(Prozess p) {
         }
     }
 
+    private void rewriteProcessMetadata(List<Prozess> processes) {
+        String currentProcessTitele = null;
```

Comment:
You must not initialise this property `currentProcessTitele` (contains even a spelling error) as this property will be overwritten in any iteration of the loop.

Reply:
I must, otherwise the Java compiler complains. This is because—in abstract terms—method `process.getTitel()` might already throw an exception, and then the variable would not have been initialized at the `!= null` check. In fact, a variable in Java could always be implicitly initialized with `null`, but that is the educational part of Java. But that's what you only understand today if you've worked with such historical languages as C before.
Thanks for the typo, I have corrected.

Reply:
“Titele”—that was Swabian. :wink: